runtime/queries/column_timeseries.go

```diff
@@ -410,7 +419,7 @@ func getFilterFromDimensionValuesFilter(
 				if lv.GetKind() == nil {
 					continue
 				}
-				likeClause += escapedName + " " + prefix + " ILIKE '" + EscapeSingleQuotes(lv.GetStringValue()) + "'"
+				likeClause += escapedName + " " + prefix + " ILIKE '" + escapeSingleQuotes(lv.GetStringValue()) + "'"
```

Comment:
confused, it should be string, putting a number here seems unreasonable, what will be the purpose?

Reply:
You're right, it doesn't make sense – let's add a check to ensure it's a string (and return an error otherwise), and also use `?` and args for the string value